README.md

```diff
@@ -429,6 +430,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**FlorisBoard**](https://github.com/florisboard/florisboard) <sup>**[[F-Droid](https://f-droid.org/app/dev.patrickgold.florisboard)]**</sup>
 * [**Fossify Keyboard**](https://github.com/FossifyOrg/Keyboard) <sup>**[[F-Droid](https://f-droid.org/app/org.fossify.keyboard)]**</sup>
 * [**FutharkBoard**](https://github.com/DrMaxNix/futharkboard) <sup>**[[F-Droid](https://f-droid.org/app/de.drmaxnix.futharkboard)]**</sup>
+* [**Futo Keyboard**](https://gitlab.futo.org/keyboard/latinime)
```

Comment:
Futo Keyboard isn't FOSS unfortunately. Please see #508 for further information.